packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
+    await this._initializationPromise;
```

Comment:
Yes, we should remove the `get queue()` method altogether and only use `this.getQueue()`.